### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ScreenReader: The heading step1 and import flat file wizard are both announced as heading 1 in browse mode.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12962]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#Benchmark;#NVDA;#AILimited;JAWS;#AINotInScope;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3,JAWS

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Launch screen reader as NVDA and navigate to heading step1 and heading "import flat file wizard" in browse mode insert +7 from drop down "server the database is in"

**Actual:**
On navigating to heading "step1" and heading "import flat file wizard" in browse mode insert +7 from drop down "server the database is in" both the headings are announced as heading 1.

**Issue observed with JAWS**

**Same issue observed in page:**

1. 1. Import Extension - Preview Data
2. Import Extension - Modify Columns
3. Import Extension - Summary
4.   Notebooks Python Dependencies Wizard 

**Expected:**
 Heading "import flat file wizard" should be heading 1 and heading "step1" should be heading 2 thus making heading "specify input file" heading 3

**User Impact:**
The screen reader users will get the wrong page structure.

**Attachment:**
[#12962_A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/5401641/12962_A11y_SQL.Azure.Data.Studio_Import.Extension.-.Specify.Input.File_ScreenReader.zip)

**Comment by fsteffi** [comment]
hi @alanrenmsft has the item "import wizard " been removed
![image](https://user-images.githubusercontent.com/24580570/220045177-6de6395e-d87e-4874-a15e-6f65705fe78e.png)


GithubTags:#Regressed:02-20-23;

**Comment by fsteffi** [comment]
issue fixed
Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

GithubTags:#Closed;

[Screen Recording (2).webm](https://user-images.githubusercontent.com/24580570/220857867-9437801c-0d4b-41e0-af2a-f0a3b76f97b8.webm)
